public/app/features/plugins/admin/components/PluginDetailsRightPanel.tsx

```diff
@@ -12,13 +12,14 @@ import { CatalogPlugin } from '../types';
 type Props = {
   info: PageInfoItem[];
   plugin: CatalogPlugin;
+  narrowScreen?: boolean;
```

Comment:
this should be called `width` and have a default value of 250px and in the special cases of narrowScreen it can be passed as `auto`